openfe/protocols/openmm_septop/equil_septop_settings.py

```diff
@@ -301,6 +301,17 @@ class SepTopEquilOutputSettings(MDOutputSettings):
     """
 
 
+class ComplexEquilOutputSettings(SepTopEquilOutputSettings):
```

Comment:
Given the default is encouraged to be `all` for both solvent and complex, why not just force all for both? That will make it less complicated overall.

Otherwise, I would try to align the names - one is called SepTopEquilOutputSetitngs and the other is ComplexEquilOutputSettings, that's somewhat confusing naming wise.